SCons/Taskmaster/JobTests.py

```diff
@@ -284,50 +285,6 @@ def runTest(self):
         self.assertFalse(taskmaster.num_failed,
                     "some task(s) failed to execute")
 
-        # Verify that parallel jobs will pull all of the completed tasks
```

Comment:
Not sure we should delete this quite yet if we're still providing legacy_sched?
@mwichmann  thoughts?

Reply:
Yeah, I was a little fed up with this test and maybe overreacted a bit and just deleted it. But every time I've experimented with scheduler stuff it got in the way. Is there a way to temporarily opt-in to `--experimental=legacy_sched` here which would make the test pass?

Reply:
@bdbaddog - What do you think? Restore the test but force it to `legacy_sched` somehow? Or do away with it. It really is testing a behavior of the legacy scheduler, so I don't see that there is much value in adapting it for the new scheduler.

Reply:
Yes. restore. There's got to be some prior art in the test suite about forcing a GetOption(), maybe calling SetOption() before in the test before the rest is run?

Reply:
I figured out how to do it, quite easy. Restored.